main.go

```diff
@@ -178,6 +179,14 @@ func main() {
 		setupLog.Error(err, "unable to create controller", "controller", "StorageClassClaim")
 		os.Exit(1)
 	}
+	if err = (&storageclassclaimsecret.StorageClassClaimSecretReconciler{
+		Client:            mgr.GetClient(),
```

Comment:
Need to do similar cache change here https://github.com/red-hat-storage/ocs-operator/pull/1811

Reply:
Done